pubspec.yaml

```
@@ -151,4 +191,6 @@ flutter_launcher_icons:
   #         weight: 700
   #
   # For details regarding fonts from package dependencies,
-  # see https://flutter.dev/to/font-from-package
\ No newline at end of file
+  # see https://flutter.dev/to/font-from-package
+
+  
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)